shotover-proxy/src/transforms/redis/sink_cluster.rs

```diff
@@ -495,7 +495,7 @@ impl RedisSinkCluster {
         warn!("Could not route request - short circuiting");
         if let Err(e) = self.send_error_response(
             one_tx,
-            "ERR Shotover RedisSinkCluster does not not support this command used in this way",
+            "ERR unknown command - Shotover RedisSinkCluster does not not support this command",
```

Comment:
Should this be `unsupported command` instead?

Reply:
[`unknown` is actually necessary.](https://github.com/lettuce-io/lettuce-core/blob/main/src/main/java/io/lettuce/core/RedisHandshake.java#L262)

Reply:
Ah, thanks for that.